.github/workflows/build.yml

```diff
@@ -20,14 +26,16 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install -U pytest
```

Comment:
@Quantalabs what does -U do?

Reply:
It will upgrade to the latest version. Check the [docs](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-U) for more.However, why are we installing pytest if our tests are not compatible with discovery?

Reply:
Pytest is there for code coverage and may be used later. Since it doesn't affect the merge, we'll ignore it